debian/.gitignore

```diff
@@ -0,0 +1,9 @@
+matrix-synapse.*.debhelper
+matrix-synapse.debhelper.log
+matrix-synapse.substvars
+matrix-synapse/
+matrix-synapse-*/
+files
+debhelper-build-stamp
+source/
```

Comment:
shouldn't this be in vcs?

Reply:
no, because it's loaded from the mounted dir and is purely temporary while building.

Reply:
what do you mean by "the mounted dir"?

Reply:
the synapse repo is mounted into the docker image, and as part of building, it copies it from the mounted source dir into the debian/ dir to work on as a copy.